.golangci.yaml

```diff
@@ -8,8 +8,10 @@ linters-settings:
     values:
       const:
        - LICENSE: Apache-2.0
+      regexp:
+       YEAR: 20\d*-*\d*
```

Comment:
This may be going too deep, but should the regex be `20\d*-20\d*`?

A 20xx - 19xx range is invalid since it's backwards, and I doubt we'll need to worry about 100-1000 years in the future :)

I'm fine with leaving it so I gave an approval, but wanted to point this out, just in case.

Reply:
>  doubt we'll need to worry about 100-1000 years in the future

Tanzu is beyond time Nolan 🤯 

but true, using `20\d*-20\d*` would at least give us today (`2022`) through `2099`

Reply:
@stmcginnis are you able to update this to the more sensible limit such that we can catch edge-case typos in dates.

Reply:
Done!